### DISCUSSION TITLE
The `url` supplied for the path (./dev/*/*) repository does not exist

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#17160] [Q&A / Support]
Hi Guys, after installing docker based Pimcore, as soon as I want to install a package (of course in the container) I get the following error message:
"The `url` supplied for the path (./dev/*/*) repository does not exist." :/

**Comment by ghost** [comment]
dev folder is missing ok but why ^^

**Comment by kingjia90** [comment]
Hello,
which package? so far seems that wherever it is used, there is a folder in there.
eg. In the demo, the folder is there https://github.com/pimcore/demo/tree/10.3/dev but in the skeleton there is no equivalent of ["url": "./dev/*/*"](https://github.com/pimcore/demo/blob/8bcf5b4906cfcf65c832462532cd10a1d7af2433/composer.json#L68) in https://github.com/pimcore/skeleton/blob/11.x/composer.json
